src/lib/jsonld/work.js

```diff
@@ -182,22 +248,44 @@ function getMovie({ id, title, description, url, cover, manifestations = [] }) {
  * @returns {object} JSON-LD representation of work
  */
 export function getJSONLD(work) {
-  const url = getCanonicalWorkUrl(work);
-
+  // set parameters for getCanonicalUrl method
```

Comment:
Hvis JsonLd IKKE er server-side-ting, hvilket jeg tror det er, så kan man overveje nedenstående. Hvis JsonLd kun bruges server-side, så bare ignorér det der står under.

Hvis denne var et hook (og hed f.eks. "useJSONLD"), så ville man kunne give den en `workId`, og så kunne den selv sørge for sine data, og derved ville man kunne genbruge den alle steder uden at skulle tænke på hvilken `work` man fodrer den med :grinning: